### PR TITLE
Added support for GitHub

### PR DESCRIPTION
**Pull request by ChrisPates**
# Contributing to Customizations for AWS Control Tower (CfCT).

Closes: #198 

Thank you for your interest in contributing to Customizations for AWS Control Tower (CfCT).

At this time, we are not accepting contributions. If contributions are accepted in the future, Customizations for AWS Control Tower (CfCT) is released under the [Apache license](http://aws.amazon.com/apache2.0/) and any code submitted will be released under that license.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,15 +53,57 @@ chmod +x ./deployment/build-s3-dist.sh
...
+* Connect to GitHub
+  * Select **Install a new app**
+  * Select the GitHub User/Organization for your repository
+* AWS Connector for GitHub
+  * Under **Repository access**, select **Only select repositories** and select only the repository you created earlier.
+  * Select **Save**
+* Make a note of the Code Connections ARN, as you'll need to provide this when deploying the AWS CloudFormation stack.
+
+* Get the link of the custom-control-tower-initiation.template in the root of your repository.
+* Deploy the Customizations for AWS Control Tower solution to your account by launching a new AWS CloudFormation stack using the link of the custom-control-tower-initiation.template.
```
Comment (kkvinjam) [inline README.md:93]: Consider removing `using the link of the` . Unclear reference on which link IMO.

File: README.md [hunk 2]
```diff
@@ -49,15 +53,57 @@ chmod +x ./deployment/build-s3-dist.sh
...
+  * Select `GitHub` as the **provider**
+  * **Create GitHub App connection** in **Connection name** type `GitHub - CfCT`
+  * Select **Connect to GitHub**
+* Connect to GitHub
+  * Select **Install a new app**
+  * Select the GitHub User/Organization for your repository
+* AWS Connector for GitHub
+  * Under **Repository access**, select **Only select repositories** and select only the repository you created earlier.
+  * Select **Save**
+* Make a note of the Code Connections ARN, as you'll need to provide this when deploying the AWS CloudFormation stack.
```
Comment (kkvinjam) [inline README.md:90]: consider 'Note the Code Connections ARN - you'll need it when deploying the CloudFormation stack.'

File: README.md [hunk 3]
```diff
@@ -49,15 +53,57 @@ chmod +x ./deployment/build-s3-dist.sh
...
+
+* Get the link of the custom-control-tower-initiation.template in the root of your repository.
```
Comment (kkvinjam) [inline README.md:92]: from the root

File: README.md [hunk 4]
```diff
@@ -49,15 +53,57 @@ chmod +x ./deployment/build-s3-dist.sh
...
 
-## Deploying the customized solution
+### Deploying the customized solution
 * Get the link of the custom-control-tower-initiation.template loaded to your Amazon S3 bucket. 
 * Deploy the Customizations for AWS Control Tower solution to your account by launching a new AWS CloudFormation stack using the link of the custom-control-tower-initiation.template.
 
+## Deploying with GitHub as the Source
+
+### Prepare a GitHub Repository
+Create a repository within your GitHub account, and populate with the contents of this repository.
```
Comment (kkvinjam) [inline README.md:63]: Why do they need  complete content of this repository? Isn't this GitHub repo meant for source of customizations?

**Review by kkvinjam — CHANGES_REQUESTED** [review]
Validated the code in one of control tower accounts. Included few suggestions to the instructions.

File: README.md (hunk 4 above)
Comment (ChrisPates) [reply README.md]: Good point, updated.

**Pull request closed without merging**